### PR TITLE
Changelog+api tweaks

### PR DESCRIPTION
**Pull request by bc-mkatz**
Changelog content + heading-CSS tweaks. `_api.md` [root] copy edits + link restoration. Link shortening in `_api_root_oauth.md` to restore proper scrolling to targets.

### CODE REVIEW

File: source/api/v2/index.html.md [hunk 1]
```diff
@@ -30,6 +30,44 @@ includes:
   - api_rate_limits_basic
   - api_guides_curl_quickstart
   - api_guides_oauth_transition
+  - api_objects_blog_post.md
+  - api_CRUD_blog_posts.md
+  - api_CRUD_blog_tags.md
+  - api_objects_redirect.md
+  - api_CRUD_redirects.md
```
Comment (bc-AlyssNoland) [inline source/api/v2/index.html.md]: We don't need to include .md. That should be dropped

File: source/includes/_api.md [hunk 2]
```diff
@@ -1,47 +1,49 @@
...
 
 Once approved, you will receive one or more emails containing:
 
-* **Your Partner ID**: required to submit an app for [App Store][49] consideration
-* **Temporary Partner Portal credentials**: change your password immediately after logging in
+* **Your Partner ID**: required to submit an app for [App Store](https://www.bigcommerce.com/apps/) consideration.
+* **Temporary Partner Portal credentials**: change your password immediately after logging in.
 
-NOTE: If you do not receive your Partner ID by email, contact [appstore@bigcommerce.com][50].
+NOTE: If you do not receive your Partner ID by email, contact [appstore@bigcommerce.com](mailto:appstore@bigcommerce.com).
```
Comment (bc-AlyssNoland) [inline source/includes/_api.md]: I believe this needs to be changed to partnersupport@bigcommerce.com 

@jcnich

File: source/includes/_api_CRUD_orders.md [hunk 3]
```diff
@@ -532,25 +543,30 @@ The following properties of the order are read-only. If one or more of these pro
 *   shipping_address_count
 *   coupons
 
-### <span class="jumptarget"> Notes </span>
+<aside class="warning">
+<span class="aside-warning-hd">Migration Note</span><br><br>
+All "Notes," "KNOWN ISSUES," and "PRO TIPs" formatted as `>blockquote` in this `_api_CRUD_orders.md` file – including those directly below – must be reformatted as appropriately-colored `asides`.
```
Comment (bc-AlyssNoland) [inline source/includes/_api_CRUD_orders.md]: What is the point of this note? It doesn't seem relevant to developers

File: source/includes/_api_objects_order_coupon.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
-# <span class="jumptarget"> Order Coupons </span>
+## <span class="jumptarget"> Order Coupons </span>
 
-Coupon code applied to an order
+Coupon codes applied to an order.
```
Comment (bc-AlyssNoland) [inline source/includes/_api_objects_order_coupon.md]: Only a single coupon code can be applied to any given order. Might be better to use language that indicates this.

File: source/includes/_api.md (hunk 2 above)
Comment (jcnich) [inline source/includes/_api.md]: Yep correct, should be partnersupport@bigcommerce.com. We might move to remove this line in favor of other instructions soon, but I'll get with Katz on that when any updates are ready from my end.

File: source/includes/api_overview.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+## Build best-of-breed solutions for fast-growing online businesses
+
+The Bigcommerce Stores API features a RESTful architecture, allowing you to code in the language of your choice. All connections require authentication and are secured by TLS encryption. It supports [the JSON media type](/api/media-types) and uses UTF-8 character encoding.
+
+With clever use of this API, you can automate various commerce, business, and publishing tasks and integrate all kinds of apps with our platform.
+
+## Let's get started!
+
+### 1\. Join the Technology Partner Program
```
Comment (bc-AlyssNoland) [inline source/includes/api_overview.md]: Does this 1. need escaping or is the \ character random?

File: source/stylesheets/_custom.scss [hunk 6]
```diff
@@ -133,7 +133,9 @@
 
   h3, h4 {
     padding-top: 2%;
-    padding-bottom: 2%;
```
Comment (bc-AlyssNoland) [inline source/stylesheets/_custom.scss]: Might be better to change to 0.5% or something smaller. I can't remember why I did this, but I'd check through changelog and some other areas to ensure it didn't have a negative impact.

File: source/includes/_api_CRUD_orders.md (hunk 3 above)
Comment (bc-mkatz) [inline source/includes/_api_CRUD_orders.md]: As we discussed: Temporary notes-to-self, as reminders of needed changes. Will be resolved/removed before we publish.

File: source/includes/_api.md (hunk 2 above)
Comment (bc-mkatz) [inline source/includes/_api.md]: Done, thanks.

File: source/includes/_api_objects_order_coupon.md (hunk 4 above)
Comment (bc-mkatz) [inline source/includes/_api_objects_order_coupon.md]: Rolled back to singular.

File: source/includes/api_overview.md (hunk 5 above)
Comment (bc-mkatz) [inline source/includes/api_overview.md]: This file isn't included/used anywhere, and will not be. I placed it in the `/includes/` subdirectory only as a source for backfilling/copying "reference-style" links that got orphaned/broken during initial scrape from live Dev Portal.

File: source/stylesheets/_custom.scss (hunk 6 above)
Comment (bc-mkatz) [inline source/stylesheets/_custom.scss]: Tried 0.5%. The difference (between that and the implicit 0% that I pushed) is nearly negligible. But at least to my eye, 0% looks slightly cleaner – that keeps each heading closer to its body text, which feels like a smoother flow from heading to body. Suggest we merge as-is, then you can tweak later if you'd prefer more leading there.
